devices/power.py

```diff
@@ -330,6 +332,45 @@ def reset(self):
         time.sleep(5)
         self.on()
 
+class Ip9820(PowerDevice):
+    def __init__(self, ip_address, port, username="admin", password="12345678"):
+        PowerDevice.__init__(self, ip_address, username, password)
+        self._ip_address = ip_address
+        self.port = port
+
+        # create a password manager
+        password_mgr = urllib2.HTTPPasswordMgrWithDefaultRealm()
+        password_mgr.add_password(None, 'http://' + ip_address, username, password)
+        handler = urllib2.HTTPBasicAuthHandler(password_mgr)
+        opener = urllib2.build_opener(handler)
+        # Now all calls to urllib2.urlopen use our opener.
+        urllib2.install_opener(opener)
+
+    def on(self):
+        print("Power On Port(%s)\n" % self.port)
+        return urllib2.urlopen('http://' + self._ip_address + '/set.cmd?cmd=setpower+p6' + str(self.port) + '=1')
+
+    def off(self):
+        print("Power Off Port(%s)\n" % self.port)
+        return urllib2.urlopen('http://' + self._ip_address + '/set.cmd?cmd=setpower+p6' + str(self.port) + '=0')
+
+    def reset(self):
+        self.off()
+        time.sleep(5)
+        self.on()
+
+    def getPowerStatus(self):
+
+        oid = self.oid_Outlet + '.' + str(self.port) + '.0'
```

Comment:
Where does self.oid_Outlet get set? Please just remove these functions if they were used for testing.

Reply:
Will remove it, thanks.